proto/wechaty/puppet/event.proto

```diff
@@ -24,6 +24,12 @@ enum EventType {
   EVENT_TYPE_RESET       = 24;
   EVENT_TYPE_LOGIN       = 25;
   EVENT_TYPE_LOGOUT      = 26;
+
+  EVENT_TYPE_CONTACT_DIRTY = 27;
+  EVENT_TYPE_ROOM_DIRTY = 28;
+  EVENT_TYPE_FRIENDSHIP_DIRTY = 29;
+  EVENT_TYPE_MESSAGE_DIRTY = 30;
+  EVENT_TYPE_ROOM_MEMBER_DIRTY = 31;
```

Comment:
Is there a must to separate dirty `ROOM` and `ROOM_MEMBER`?

I feel they can be merged to one event?